ECIP-VeriBlock-PoP-51%-Protection.md

```diff
@@ -0,0 +1,575 @@
+---
+lang: en
+ecip: ECIP 1094
+title: VeriBlock Proof-of-Proof 51%-Attack Prevention
+category: core
+status: Draft
+type: Standards Track
+discussions-to: https://github.com/max-sanchez/ECIPs-1/issues/?
```

Comment:
Uh, this requires some valid link. Easiest would be to create an issue here on the repository.